bedrock/mozorg/middleware.py

```diff
@@ -45,3 +46,17 @@ def process_view(self, request, view, view_args, view_kwargs):
         else:
             f = super(MozorgRequestTimingMiddleware, self)
             f.process_view(request, view, view_args, view_kwargs)
+
+
+class CrossOriginResourceSharingMiddleware(object):
```

Comment:
I originally considered using an app like that (that app in particular too), and decided against it for some reasons that may not be so valid now that I'm sort've doing what the app does anyway. I've glanced through the code and it looks good, I think I might just adapt their app instead.